src/org/labkey/test/tests/KnitrReportTest.java

```diff
@@ -199,13 +199,10 @@ private void verifyAdhocReportDependencies(String viewName, String dependencies)
 
         createKnitrReport(rmdDependenciesReport, RReportHelper.ReportOption.knitrMarkdown);
 
-        pauseJsErrorChecker(); // Don't fail due to "$ is not a function"
-        {
-            _rReportHelper.clickReportTab();
-            waitForElement(Locator.id("mtcars_table"));
-            assertElementNotPresent(Locator.id("mtcars_table_wrapper")); // Created by jQuery
-        }
-        resumeJsErrorChecker();
+        _rReportHelper.clickReportTab();
+        waitForElement(Locator.id("mtcars_table"));
+        assertElementNotPresent(Locator.id("mtcars_table_wrapper")); // Created by jQuery
+        checkExpectedErrors(1); // JavaScript error: "$(...).dataTable is not a function"
```

Comment:
Use `int errorCountBefore = getServerErrorCount();`  and `checkExpectedErrors(errorCountBefore + 1);` ?

Reply:
Done.